agent/src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -1123,7 +1123,11 @@ bool main_thread::handle_arp_monitor()
     auto iIfaceIndex = arp_notif->params().iface_idx;
 
     char if_name_buffer[IF_NAMESIZE];
-    if (!if_indextoname(iIfaceIndex, if_name_buffer)) {
+
+    auto iface_index_name = network_utils::linux_get_iface_name(iIfaceIndex);
+    std::copy_n(iface_index_name.begin(), iface_index_name.length(), if_name_buffer);
+
+    if (!if_name_buffer[0]) {
         LOG(ERROR) << "Failed to find iface of iface_index" << int(iIfaceIndex);
         return false;
     }
```

Comment:
Here also, remove `if_name_buffer` declaration, and put `linux_get_iface_name()` output directly on `strIfaceName`.

Reply:
fixed